packages/@react-spectrum/tabs/src/tabs.css

```diff
@@ -17,9 +17,45 @@
   height: 100%;
 }
 
+.react-spectrum-Tabs--collapsible {
```

Comment:
should any of this be making its way into the actual css?

Reply:
ya, I prob will move it into the spectrum css (barring any other opinions) just kept it in this format since it is pretty much copypaste from v2 and this tabs.css file already existed